docs/index.md

```diff
@@ -3,12 +3,24 @@
 page_title: "clickhouse Provider"
 subcategory: ""
 description: |-
-  
+  This is the official provider for ClickHouse Cloud.
+  With this provider you can deploy a ClickHouse instance on AWS, Google Cloud or Azure Cloud.
```

Comment:
This sounds like I can provision on-premise, (yeah it doesn't make sense) not a CH managed service. Maybe this would work?
```suggestion
  With this provider you can deploy a ClickHouse Cloud instance on AWS, Google Cloud or Azure Cloud.
```